### PR TITLE
tests: fix incompatibility with pytest>=2.8.0

### PR DESCRIPTION
**Pull request by SamiHiltunen**
* FIX Removes calls to PluginManager
  consider_setuptools_entrypoints() removed in PyTest 2.8.0.

Signed-off-by: Sami Hiltunen <sami.mikael.hiltunen@cern.ch>

### CODE REVIEW

**Comment by jirikuncar** [comment]
- [ ] add `coveralls` to `.travis.yml`
- [ ] rebase needed

**Comment by SamiHiltunen** [comment]
@jirikuncar done

**Comment by jirikuncar** [comment]
@SamiHiltunen looking at last failing tests you might need to bump `invenio-base` version.

File: .travis.invenio.cfg [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+SECRET_KEY = 'MY_SECRET'
+
+# Disable all automatic asset building - false is /usr/bin/false.
+ASSETS_AUTO_BUILD = False
+
+PACKAGES = [
+    'invenio_accounts',
+    'invenio_upgrader',
+    'invenio_testing',
+    'invenio_ext',
```
Comment (jirikuncar) [inline .travis.invenio.cfg]: ext and utils should not be in packages ...

**Comment by SamiHiltunen** [comment]
If you need this for old release, please ping me.

**Pull request closed without merging**